configure.ac

```diff
@@ -107,7 +108,7 @@ if test x"$enable_liblightdm_qt4" != "xno" ; then
     fi
     if test x"$MOC4" = xmoc; then
         if test x"$(readlink $(which $MOC4))" = xqtchooser; then
-            MOC4="$MOC4 --qt=qt4"
+            MOC4="$MOC4 --qt=4"
```

Comment:
Is this not working somewhere? I get the following (which suggests 4 and 5 are the recommended value to be passed).

$ qtchooser -l
4
5
default
qt4-x86_64-linux-gnu
qt4
qt5-x86_64-linux-gnu
qt5

Reply:
This was a downstream patch we carried in Fedora / EPEL since some long time.  The patch actually changes to an invocation with the recommend values, as you described.